EscelForce.Business.Login/Interfaces/IUpdateExtractionMapService.cs

```diff
@@ -34,5 +34,6 @@ public interface IUpdateExtractionMapService
         ServiceResponseModel<bool> SubmitPreviousFieldSelection();
 
         ServiceResponseModel<bool> DeleteSelectedChild(string childName);
+        void clear();
```

Comment:
Please map the response of this method to `ServiceResponseModel<bool>` to maintain consistency.

Reply:
@ritwiksen  Done.Please check